tgui/packages/tgui/interfaces/MarionetteRemote.js

```diff
@@ -0,0 +1,163 @@
+import { useBackend } from '../backend';
+import { Window } from '../layouts';
+import { Button, Flex, LabeledList, Section, Stack } from '../components';
+
+export const MarionetteRemote = (_props, context) => {
+  const { act, data } = useBackend(context);
+  const { entered_data, selected_command, implants } = data;
+
+  return (
+    <Window
+      title="Marionette Remote"
+      width={410}
+      height={550}
+      theme="syndicate" >
+      <Window.Content scrollable>
+        <Stack vertical fill minHeight="1%" maxHeight="100%">
```

Comment:
minHeight 1% seems hacky, consider putting comments explaining why it's necessary.